### DISCUSSION TITLE
LovyanGFX supported CVBS

### DISCUSSION THREAD

**Discussion by riraosan** [Roger-random/ESP_8_BIT_composite#37] [Show and tell]
Hi @Roger-random みなさん。

LovyanGFXの作者、@lovyan03氏があなたの成果を発展させて高速描画ライブラリを使ったCVBS描画を実現されたようです。もしよろしければ参照ください。

https://github.com/lovyan03/LovyanGFX/blob/develop/src/lgfx/v1/platforms/esp32/Panel_CVBS.cpp

簡単な使い方はこちらを、参照してください。

https://github.com/lovyan03/LovyanGFX/blob/develop/doc/Panel_CVBS.md

**Comment by Roger-random** [comment]
Thank you for letting me know! I have added a new "Alternatives" section to my README, where I list LovyanGFX as well as FabGL and Bitluni's library. (Commit https://github.com/Roger-random/ESP_8_BIT_composite/commit/39f1b6410df4556283dd803bfeb8b5dd1e4cd1e8)

**Reply by riraosan** [reply]
Thank you.

**Comment by tobozo** [comment]
hi ^^ 

Sorry for intruding, I'm afraid linking to the developer branch from the ReadMe will only end up in generating unnecessary traffic on LovyanGFX issue tracker, and create a situation where renaming that branch or moving it to a private repository will be the only choices to keep a peace of mind and continue to provide quality work.

Meanwhile it is probably more appropriate to add conditional includes based on the LGFX version in your project, and wait for the next release of LovyanGFX to enable the feature.

See the [gitTagVersion.h](https://github.com/lovyan03/LovyanGFX/blob/master/src/lgfx/v1/gitTagVersion.h) you can use to ease that integration.

be well

tobozo

**Reply by riraosan** [reply]
@tobozo 

Well, but I would like to separate your individuals and mine. If I were to share my development branch for some purpose by someone somewhere, for the purpose of trying to contribute to some purpose by someone somewhere, it would not be my individuals, and I would remain quiet.
However, I am sorry that I shared it without the author's consent.
Also, I will not share public branches that are under development.

**Comment by Roger-random** [comment]
LovyanGFV CVBS support has not yet reached main branch, but it appears M5Stack has already released hardware for it. SKU #M125
https://shop.m5stack.com/products/rca-audio-video-composite-module-13-2

(Assuming I read the LGFX usage in example code correctly... https://github.com/m5stack/M5Stack/blob/master/examples/Unit/RCA/RCA.ino)
